### ISSUE TITLE
Translation not working on some properties (.resx files)

### ISSUE THREAD

**Issue by TomsDev67** [leocb/MaterialSkin#66]
Hello 😃 

My project is translated in french and english. I'm using the Winform designer to edit and add translation. Some properties in some components are not triggered when I'm editing a properties, the result is that translation and adjustement are not working and it's modyfing the default layer. See below

Steps : 

### 1. Select your main Form. You'll find in properties list the property called "Language"
![0](https://user-images.githubusercontent.com/8772453/75758851-e632e880-5d34-11ea-8a5a-28d20912aabc.png)

### 2. Modify it to 'french' for example.
![3](https://user-images.githubusercontent.com/8772453/75758923-006cc680-5d35-11ea-8df9-cbf8c76959b5.PNG)

### 3. Now modify texts, move components, do what you want in your UI you want to be changed when you're in french.
![2](https://user-images.githubusercontent.com/8772453/75759047-35791900-5d35-11ea-98e8-3508481e7a08.PNG)

### 4. Save, you'll see a new **resx** file added to your form for the language you've selected
![1](https://user-images.githubusercontent.com/8772453/75759091-49247f80-5d35-11ea-9f1f-f3804b58a6fa.PNG)

### 5. Change back the language to default one
![5](https://user-images.githubusercontent.com/8772453/75759160-66f1e480-5d35-11ea-91fc-5412d49a0b28.PNG)

### 6. It should change back everything in your UI. But we can see "Hint" are not reverted
![4](https://user-images.githubusercontent.com/8772453/75759266-91dc3880-5d35-11ea-85d0-4bf23c74e20b.PNG)
 
### 7. If you open the french resx you'll only see the label text
![6](https://user-images.githubusercontent.com/8772453/75759355-b46e5180-5d35-11ea-8fb2-de4c1fcaee4d.PNG)

For now I saw this issue on : 
- **Hint**, **UseAccent**, **UseTallSize** properties on all components
- **Text** properties on TabControl

Quick search on this issue gave me this _https://stackoverflow.com/questions/26250972/net-localize-custom-control_ but not sure if it will solve this issue.

**Comment by leocb** [comment]
Hello, thanks for reporting, I haven't paid much attention the internationalization of the project yet, non-Latin languages, for example, are **very** broken due to the roboto font not supporting them (we need the noto font for those).
And yes it looks like the stack overflow link you provided is exactly the problem. The simplest solution seems to be in the second answer:

>Problem solved by adding:
>[Localizable(true)]
>to the localizable properties of the custom control.

So try adding that decorator above the `Hint` and `Text` properties on the components. Try it and  Open a PR if you will :)

**Comment by TomsDev67** [comment]
Hi,

I've tried on Textbox and ComboBox to add **Localizable(true)** but it was not really working, the issue is still the same. I was not able to find something else to made it working.
I've tried on VS2017 and VS2019 in case of.

**Comment by leocb** [comment]
Weird, it works for me, did you recompile and reloaded after you added the decorator to the lib?

**Comment by TomsDev67** [comment]
Yes :S
I've done something like this :
```
[Category("Material Skin"), DefaultValue(""), Localizable(true), Description("The displayed control title")]
public string Hint
```
Cleaned, recompiled and tried directly in your test project. But maybe something was missing ?

**Comment by leocb** [comment]
When I'm not sure you can combo that attribute with the others, try adding it to it's own attribute "array", like this:
```
[Category("Material Skin"), DefaultValue(""), Description("The displayed control title")]
[Localizable(true)]
public string....
```

**Comment by TomsDev67** [comment]
I've tried to put it in his own [...] like your example, but still the same behavior on my side 🤔 
The **Hint** property is always overriden directly in the form.Designer.cs instead of modifying resx files.

**Comment by leocb** [comment]
Hmm, maybe it's because you added the Localizable(true) to the attribute after you enabled localization on the project, maybe try adding it by hand in the designer.cs? In my case I added it before enabling localization on the project, then it worked.

**Comment by TomsDev67** [comment]
Hmmmm yes it's a way to investigate, that's possible that the order have an impact on what Visual Studio and the Designer is doing. I'll try and let you know.

It's like this answer https://stackoverflow.com/a/54399918/12524949

**Comment by TomsDev67** [comment]
All my tests have failed to made it work. It's always adding the Hint value in Designer file instead of resx files.

Tried in your project, in another project using a custom NuGet package, with and without having Localization property to true on Form when adding my TextBox on it.

It could be nice to have feedback from other people to compare what they've and maybe see the difference between your and my tests 8-/

And I've tried in 4.6.1 and 4.7.2 just to be sure that's not like the bug #67 😆
